src/domUpdates.js

```diff
@@ -56,7 +56,7 @@ searchBar.addEventListener('keyup', function (e) {
   if (e.keyCode === 13) {
     searchAllRecipes(recipeRepository.recipeData);
   }
-})
+});
```

Comment:
We just went through and got rid of spaces, add semi-colons, the standard dotting of the i's and crossing the t's.